packages/extension-api/src/extension-api.d.ts

```diff
@@ -788,6 +789,79 @@ declare module '@podman-desktop/api' {
     connection: ContainerProviderConnection;
   }
 
+  /**
+   * Callback for openning shell session
+   */
+  export interface ProviderConnectionShellAccess {
+    /**
+     * Opens new session using ProviderConnectionShellAccessImpl class
+     * @example
+     * const providerConnectionShellAccess = new ProviderConnectionShellAccessImpl(machineInfo);
```

Comment:
Example need to be in triple quote, and preferably with the language associate

https://github.com/containers/podman-desktop/blob/6c8c434fa61f175288c33e9b0ae043119166783a/packages/extension-api/src/extension-api.d.ts#L704-L705

Reply:
@axel7083  Needs to be inside the ```typescript valid typescript code? thinking about this => `session.onData(data => {...`

Reply:
I am not sure to understand what you mean

Reply:
thanks